manifests/job.pp

```diff
@@ -34,6 +37,7 @@
   $jobname  = $title,
   $enabled  = 1,
   $ensure   = 'present',
+  $folder   = undef,
```

Comment:
I added a link to the Folder plugin in the description.

In short:  the Folder Plugin allows you to organise jobs in "folders".

Traditionally you'd organise jobs in views, but if you have a lot of jobs and views, things will get very cluttered.

It's a Cloudbees supported plugin.